src/ReactWrapper.js

```diff
@@ -468,18 +468,18 @@ export default class ReactWrapper {
    * testing events should be met with some skepticism.
    *
    * @param {String} event
-   * @param {Array} args
+   * @param {Object} mock (optional)
    * @returns {ReactWrapper}
    */
-  simulate(event, ...args) {
+  simulate(event, mock = {}) {
     this.single(n => {
       const mappedEvent = mapNativeEventNames(event);
       const eventFn = Simulate[mappedEvent];
       if (!eventFn) {
         throw new TypeError(`ReactWrapper::simulate() event '${event}' does not exist`);
       }
 
-      eventFn(findDOMNode(n), ...args);
+      eventFn(findDOMNode(n), mock);
```

Comment:
@blainekasten `makeSimulator` returns a function that takes two arguments

```js
function makeSimulator(eventType) {
  return function(domComponentOrNode, eventData) {
  ...
```

[src/test/ReactTestUtils.js#L497](https://github.com/facebook/react/blob/ba9b985406ff56c2dea192b119328b6096895097/src/test/ReactTestUtils.js#L497)

I'll have to check previous releases of React to see if it's any different